### PR TITLE
Udemy phase #2

### PR DESCRIPTION
**Pull request by waelelsafty07**

### CODE REVIEW

File: ratingStar.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+export const ratingStar = (rating, students) => {
+  const totalStar = 5;
+  const totalStarFill = parseInt((rating / 10) * 10);
+  const div = document.createElement("div");
+  div.classList.add("card-rating");
+  const mins = totalStar - rating;
+  const floatnum = (mins % 1).toFixed(4);
+  console.log(floatnum);
```
Comment (shereen-fathy) [inline ratingStar.js]: please remove log messages

File: ratingStar.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  const totalStarUnFill = parseInt((mins / 10) * 10);
+  //   console.log(totalStarUnFill);
+  //   console.log(mins);
```
Comment (shereen-fathy) [inline ratingStar.js]: delete commented code

File: ratingStar.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    div.appendChild(span);
+  }
+  for (let i = 1; i <= totalStarUnFill; i++) {
+    const span = document.createElement("span");
+    const icon = document.createElement("i");
+    icon.classList.add("fa");
+    icon.classList.add("fa-star-o");
+    span.appendChild(icon);
+    div.appendChild(span);
+  }
```
Comment (shereen-fathy) [inline ratingStar.js:22]: there is repeated code here, could you reduce it and make it cleaner?

File: readme.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+### intall jspn server
+
+npm install -g json-server
+
+### run server
+
+json-server --watch db.json
```
Comment (shereen-fathy) [inline readme.md:7]: Thank you for updating readme file 💯

**Review by shereen-fathy — CHANGES_REQUESTED** [review]

**Review by shereen-fathy — COMMENTED** [review]
search icon on small screen is not working and I know its not in the requirements so I think it’s better to remove it

**Comment by waelelsafty07** [comment]
i finish all ur comments u can check it now @shereen-fathy

**Review by shereen-fathy — APPROVED** [review]
@waelelsafty07 Great work, Thank you! 🚀

**Pull request merged**